Formula/buku.rb

```diff
@@ -152,5 +274,24 @@ def install
     result = shell_output("#{bin}/buku --np --sany Homebrew")
     assert_match "https://github.com/Homebrew/brew", result
     assert_match "The missing package manager for macOS", result
+
+    # Test bukuserver
+    result = shell_output("#{bin}/bukuserver --version")
+    assert_match version.to_s, result
+
+    begin
+      port = free_port
+      pid = fork do
+        exec "#{bin}/bukuserver run --host 127.0.0.1 --port #{port} 2>&1 >/dev/null"
+      end
+      sleep 5
+
+      result = shell_output("curl -s 127.0.0.1:#{port}/api/bookmarks")
+      assert_match "https://github.com/Homebrew/brew", result
+      assert_match "The missing package manager for macOS", result
+    ensure
+      Process.kill("TERM", pid)
+      Process.wait(pid)
+    end
```

Comment:
This isn't needed, brew kills it's own test processes nowadays.

Reply:
Thanks, I have removed it.